numpy/core/src/multiarray/number.c

```diff
@@ -121,89 +121,6 @@ _PyArray_SetNumericOps(PyObject *dict)
     return 0;
 }
 
-/*NUMPY_API
```

Comment:
Would it make sense to also remove the private version `_PyArray_SetNumericOps`? Or, more specifically, move what's actually needed to the one place it seems used, in `umathmodule`.

Reply:
Probably, although I think to make it much nicer would require to modify the code-generation.  `number.c` is where the "numeric ops" struct is used, so I don't particularly mind there.

Reply:
It's used only(?) in `numbers.c` to link operator to ufunc, so filling it in `numbers.c` seems fine.  `umathmodule` triggers the filling, which seems fair, since it has the dict.